client/a8c-for-agencies/sections/agency-tier/lib/get-agency-tier-info.tsx

```diff
@@ -0,0 +1,77 @@
+import AgencyPartnerLogo from 'calypso/assets/images/a8c-for-agencies/agency-tier/agency-partner-logo-small.svg';
+import EmergingPartnerLogo from 'calypso/assets/images/a8c-for-agencies/agency-tier/emerging-partner-logo-small.svg';
+import ProAgencyPartnerLogo from 'calypso/assets/images/a8c-for-agencies/agency-tier/pro-agency-partner-logo-small.svg';
+import { preventWidows } from 'calypso/lib/formatting';
+
+const getAgencyTierInfo = (
+	agencyTier: 'emerging-partner' | 'agency-partner' | 'pro-agency-partner',
+	translate: ( key: string, args?: Record< string, unknown > ) => string
+) => {
+	let tierInfo = {
```

Comment:
Nitpicking: Instead of having a variable, maybe just return the value within the switch case.